### PR TITLE
Add #full_details method to List Recurring Details response (REST API)

### PR DESCRIPTION
**Pull request by kliuchnikau**
Take a look and let's discuss

Docs for recurring details response: https://docs.adyen.com/developers/api-reference/recurring-api#recurringdetailsresult

### CODE REVIEW

File: lib/adyen/rest/authorise_payment.rb [hunk 1]
```diff
@@ -204,6 +206,29 @@ def details
           map_response_list("recurringDetailsResult.details", mapped_attributes)
         end
 
+        # Returns a list of recurring details with all the parameters
```
Comment (kliuchnikau) [inline lib/adyen/rest/authorise_payment.rb]: See https://github.com/babbel/adyen/pull/1/files#diff-75741989efe5726df89fda84b7f0d9d5L193 for what we have in the library at the moment - very limited list of fields

File: lib/adyen/util.rb [hunk 2]
```diff
@@ -111,26 +111,27 @@ def flatten(nested_hash, prefix = "", return_hash = {})
     # @param [Hash] flattened_hash The flat hash to transform
     # @param [Hash] return_hash The new hash which will be returned (needed for recursive calls)
     # @return [Hash] A nested hash structure, using strings as key.
-    def deflatten(flattened_hash, return_hash = {})
+    def deflatten(flattened_hash, return_hash = {}, symbolize_keys = false)
```
Comment (ck3g) [inline lib/adyen/util.rb]: Introducing a new param right after hash param looks dangerous.

Comment (kliuchnikau) [reply lib/adyen/util.rb]: Removed this param - now we just return strings as a hash keys. We can stringify on the client if we need to.

**Comment by kliuchnikau** [comment]
Adyen has an issue with encoding - we cannot use this metod

**Pull request closed without merging**